test/resources/controls/aws_parallelcluster_install/ec2_udev_rules_spec.rb

```diff
@@ -0,0 +1,60 @@
+control 'common_udev_configuration' do
+  title "common udev configuration for all OSes"
+
+  describe file('/etc/udev/rules.d/52-ec2-volid.rules') do
+    it { should exist }
+    its('content') { should_not be_empty }
```

Comment:
Don't we need to test something more specific for template in order to check the correct resolution?